integration_test.go

```diff
@@ -477,3 +648,44 @@ func getContractAddressesFromContractRegistry(ethHttpUrl string) ContractAddress
 		Erc20MockStrategy:      erc20MockStrategyAddr,
 	}
 }
+
+func NewEthHttpClient(rpcAddress string) (*ethclient.Client, error) {
+	client, err := ethclient.Dial(rpcAddress)
+	if err != nil {
+		return nil, err
+	}
+
+	return client, nil
+}
+
+func createWalletForOperator(t *testing.T, privKeyHex string, ethClient *ethclient.Client) walletsdk.Wallet {
+	logger, err := logging.NewZapLogger(logging.Development)
+	if err != nil {
+		panic(err)
+	}
```

Comment:
style nit: I would replace all if-panics with the one-liner `require.NoError(t, err)` everywhere.

Reply:
https://github.com/Layr-Labs/avs-sync/pull/45/commits/6fbfcd47793dd73ae72d80d2edb9fae6a66cec1a and https://github.com/Layr-Labs/avs-sync/pull/45/commits/b60f6971b36f7004b2f5d68824bf2ecd5db174d3